### PR TITLE
[DT-61][risk:no] Add CDR config property for hasFitbitSleepData and hasSurveyConductData

### PR DESCRIPTION
**Pull request by chenchals**
Description:
---
- Added properties to workbench-api.yaml
- Updated CdrVersion associated classes
- Updated tests
- Updated `db.changelog` files with new columns for cdr_version table

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [x] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/config/cdr_config_local.json [hunk 1]
```diff
@@ -26,15 +26,15 @@
...
-      "accessTier": "registered",
       "bigqueryProject": "",
       "bigqueryDataset": "",
       "creationTime": "2018-06-06 00:00:00Z",
       "releaseNumber": 1,
       "numParticipants": 0,
       "cdrDbName": "cdr",
       "hasFitbitData": false,
-      "hasCopeSurveyData": false
+      "hasCopeSurveyData": false,
```
Comment (freemabd) [inline api/config/cdr_config_local.json:36]: These configs are starting to get bloated. We can remove any of the boolean properties from hasFitbitData, hasCopeSurveyData, hasFitbitSleepData and hasSurveyConductData that are false. They are only need to be included when they are true.

Comment (chenchals) [reply api/config/cdr_config_local.json:36]: how about other isDefault set as false? Is that an exception?

File: api/config/cdr_config_local.json [hunk 2]
```diff
@@ -61,27 +58,26 @@
       "numParticipants": 234525,
       "cdrDbName": "cdr",
       "hasFitbitData": true,
-      "hasCopeSurveyData": true
+      "hasCopeSurveyData": true,
+      "accessTier": "registered"
     },
     {
       "cdrVersionId": 4,
-      "isDefault": false,
```
Comment (freemabd) [inline api/config/cdr_config_local.json:68]: please revert. isDefault does not do a null check.

File: api/config/cdr_config_local.json [hunk 3]
```diff
@@ -96,36 +92,31 @@
       "storageBasePath": "5",
       "wgsCramManifestPath": "wgs/cram/manifest.csv",
       "wgsVcfMergedStoragePath": "wgs/vcf/merged",
-      "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample"
+      "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample",
+      "accessTier": "controlled"
     },
     {
       "cdrVersionId": 6,
-      "isDefault": false,
```
Comment (freemabd) [inline api/config/cdr_config_local.json:103]: Please revert.

**Review by freemabd — COMMENTED** [review]
Please revert and isDefault changes

File: api/config/cdr_config_local.json [hunk 4]
```diff
@@ -96,36 +93,32 @@
...
       "releaseNumber": 3,
       "numParticipants": 0,
       "cdrDbName": "cdr",
-      "hasFitbitData": false,
-      "hasCopeSurveyData": false
+      "accessTier": "registered"
     },
     {
       "cdrVersionId": 7,
-      "isDefault": false,
```
Comment (freemabd) [inline api/config/cdr_config_local.json:118]: please revert

**Review by freemabd — APPROVED** [review]
Thanks

**Pull request merged**